src/Microsoft.Health.Dicom.Web/appsettings.json

```diff
@@ -124,6 +124,9 @@
       }
     }
   },
+  "HealthCheckPublisher": {
+    "Period": "00:00:10"
```

Comment:
This is pretty aggressive, isn't it? 10 seconds? Or because this is OSS, we won't worry about CMK and so it can be a lot shorter? Still, I think like `00:01:00` or at a minimum `00:00:30` is not unreasonable

Reply:
Spoke offline - 10s was chosen because of the period of k8s readiness checks, but since that is not relevant in OSS we will leave this as the default 30 seconds here. Removed the app settings